packages/cli/src/lib/cmds/play.ts

```diff
@@ -124,6 +126,49 @@ class Play {
       }
     }
 
+    // bubblewrap play --retain 86
+    if (this.args.retain) {
+      const versionToRetain = this.args.retain;
+      // Validate an integer was supplied.
+      if (!Number.isInteger(versionToRetain)) {
+        throw new Error(enUS.versionRetainedNotAnInteger);
+      }
+      if (versionToRetain > twaManifest.appVersionCode) {
+        // Cannot retain a higher version as that would take presendence.
+        const approveVersionDifference =
```

Comment:
I'm not quite sure I understand this - if we *can't* retain a higher version, then what does the user accepting the prompt do?

Reply:
Yeah good point